src/basics/numbers/is-prime/index.js

```diff
@@ -3,7 +3,20 @@
  * @returns {Boolean} `true` if `n` is a prime number, `false` otherwise.
  */
 function isPrime(n) {
-  // write your code here
+  if ( n == 1 || n == 0) {
+    return false;
+  }
+  if ( n == 2) {
+    return true ;
+  }
+
+  for  ( var i = 2 ; i < n ; i++ ) {
+      if ( n % i == 0){
+          return false ;
+          break;
+      }
+      return true ;
+  } 
```

Comment:
What will happen here if `n = 9`?

Reply:
It returns true while it should be false